themes/socialblue/socialblue.theme

```diff
@@ -32,5 +32,34 @@ function socialblue_library_info_alter(&$libraries, $extension) {
         $libraries[$key]['css']['theme'][$style_css]['weight'] = 600;
       }
     }
+
+    // Add extra JS for secondary navigation in Sky style.
+    $navbar_js = 'assets/js/navbar-secondary--sky.min.js';
+    if (file_exists($theme_path . '/' . $navbar_js)) {
```

Comment:
Isn't this always true and shouldn't this depend on whether the Sky flavour is enabled?

Reply:
Good catch - I simplified it and fixed in PR #1680